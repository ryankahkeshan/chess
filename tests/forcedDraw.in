setup
+ K a1 + k c2
+ q f5
= black
done game human human
move f5 d5
