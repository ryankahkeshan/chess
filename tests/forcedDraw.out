8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 K _ _ _ 

  abcdefgh

8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _k_ _ _
1 K _ _ _ 

  abcdefgh

8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _q_ 
4  _ _ _ _
3 _ _ _ _ 
2  _k_ _ _
1 K _ _ _ 

  abcdefgh

8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _q_ 
4  _ _ _ _
3 _ _ _ _ 
2  _k_ _ _
1 K _ _ _ 

  abcdefgh

8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _q_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _k_ _ _
1 K _ _ _ 

  abcdefgh

Stalemate!
Final Score:
White: 0.5
Black: 0.5
